adb_graphics/figures/maps.py

```diff
@@ -122,7 +133,8 @@ def draw(self):
         ''' Draw a map with political boundaries and airports only. '''
 
         self.boundaries()
-        self.draw_airports()
+        if self.model not in ['global']: # airports are too dense in global
+            self.draw_airports()
 
```

Comment:
Plotting airports creates a dense network which block out the US in larger domains, so it's skipped.